frequency-chain/frequency-xyz
# resolve svelte-fix errors

# Description
When running `npm run check` we should get no errors. Resolve all errors or as many as possible.

- Closes #105 

## Type of change


# How to Test?

run `npm run check` and see that there is only the mouse exit/enter errors. According to multiple forums, that is a known error that we can't yet solve.

# Checklist:
